apps-devstg/us-east-1/security-audit/awscloudtrail.tf

```diff
@@ -1,6 +1,6 @@
 module "cloudtrail" {
-  source                        = "github.com/binbashar/terraform-aws-cloudtrail.git?ref=0.20.1"
-  namespace                     = var.project
+  source                        = "github.com/binbashar/terraform-aws-cloudtrail.git?ref=0.21.0"
+  iamespace                     = var.project
```

Comment:
@lgallard does this variable changed from namespace to iamspace? Just wondering why this change.

Reply:
This was my bad, I'm going to fix it in next PR @exequielrafaela